src/Sulu/Bundle/Sales/OrderBundle/Order/OrderPdfManager.php

```diff
@@ -75,51 +78,54 @@ class OrderPdfManager
      * @param ObjectManager $entityManager
      * @param PdfManager $pdfManager
      * @param string $templateConfirmationPath
-     * @param string $templateDynamicPath
      * @param string $templateBasePath
      * @param string $templateHeaderPath
      * @param string $templateFooterPath
      * @param string $templateMacrosPath
      * @param string $locale
+     * @param string $templateDynamicPath
      * @param string $namePrefixDynamicOrder
      * @param string $namePrefixConfirmedOrder
      */
     public function __construct(
         ObjectManager $entityManager,
         PdfManager $pdfManager,
         $templateConfirmationPath,
-        $templateDynamicPath,
         $templateBasePath,
         $templateHeaderPath,
         $templateFooterPath,
         $templateMacrosPath,
         $locale,
-        $namePrefixDynamicOrder,
-        $namePrefixConfirmedOrder
+        $templateDynamicPath = null,
+        $namePrefixDynamicOrder = null,
+        $namePrefixConfirmedOrder = self::FALLBACK_NAME_PREFIX_ORDER_PDF
     ) {
         $this->entityManager = $entityManager;
         $this->pdfManager = $pdfManager;
         $this->templateConfirmationPath = $templateConfirmationPath;
-        $this->templateDynamicPath = $templateDynamicPath;
         $this->templateBasePath = $templateBasePath;
         $this->templateHeaderPath = $templateHeaderPath;
         $this->templateFooterPath = $templateFooterPath;
         $this->templateMacrosPath = $templateMacrosPath;
         $this->websiteLocale = $locale;
+        $this->templateDynamicPath = $templateDynamicPath;
         $this->namePrefixDynamicOrder = $namePrefixDynamicOrder;
         $this->namePrefixConfirmedOrder = $namePrefixConfirmedOrder;
     }
 
     /**
-     * @param Order $order
+     * @param $order
```

Comment:
no, sry, function gets called from a lot of different places with a lot of different Types, like suluSalesBundle/OrderBundle/Order entity or suluSalesBundle/OrderBundle/Order ApiEntity or MassivePurchaseBundle/Entity/Order ... 

And it wasnt typesafe before, i just thought it would be cool. (in the beginning)